### PR TITLE
add  0.12 release note

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

**Comment by zhujian7** [comment]
@qiujian16 for managed-serviceaccount addon:
- we added a new version v1beta1 for managed service account API: https://github.com/open-cluster-management-io/managed-serviceaccount/pull/78
- support using addon template to deploy agent: https://github.com/open-cluster-management-io/managed-serviceaccount/pull/69

**Comment by xuezhaojun** [comment]
@qiujian16 No significant upgrades in cluster-proxy

File: content/en/community/releases.md [hunk 1]
```diff
@@ -3,7 +3,31 @@ title: Releases
...
 ---
 
-Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.11.0`. Continue reading to view upcoming releases:
+Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.12.0`.
+Continue reading to view upcoming releases:
+
+## `0.12.0`, 11, Oct. 2023
+
+The Open Cluster Management team is proud to announce the release of OCM v0.12.0! We have made architecture refactor and several features in
+thie release
```
Comment (dhaiducek) [inline content/en/community/releases.md:12]:
```suggestion
The Open Cluster Management team is proud to announce the release of OCM v0.12.0! We have made architecture refactors and added several features in
this release:
```

File: content/en/community/releases.md [hunk 2]
```diff
@@ -3,7 +3,31 @@ title: Releases
...
+
+- Compoenent consolidation: we made a big code refactor to merge code in regstraiton, work, placement and registration-operator in to the `ocm` repo.
+  The original separeted code repos are currently used for maintain old release only. This code consolidation allows us to build more robust e2e tests, and build a single
+  agent binary to reduce the footprint in managed cluster.
+- Addon Template API: A new `addontemplate` API is introduced to ease the development of addons. User will not need to write code and run an addon-manager
+  controller on the hub cluster. Instead, they only need to define the `addontemplate` API to create an addon. `clusteradm` also has a new command line
+  `clusteradm addon create ...` to create an addon from resource manifests files. See more details about `addontemplate` 
+  [here](https://open-cluster-management.io/developer-guides/addon/#build-an-addon-with-addon-template)
+- Singleton agent mode: user can now choose to start agent as a single pod using the `Singleton` mode in the klusterlet.
```
Comment (dhaiducek) [inline content/en/community/releases.md:21]:
```suggestion
- Component consolidation: we made a big code refactor to merge code in registraton, work, placement and registration-operator into the [ocm](https://github.com/open-cluster-management-io/ocm) repo.
  The original separated code repos are currently used for maintaining old releases only. This code consolidation allows us to build more robust e2e tests, and build a single
  agent binary to reduce the footprint in managed clusters.
- Addon Template API: A new `addontemplate` API is introduced to ease the development of addons. Users will not need to write code and run an addon-manager
  controller on the hub cluster. Instead, they only need to define the `addontemplate` API to create an addon. `clusteradm` also has a new command
  `clusteradm addon create ...` to create an addon from resource manifests files. See more details about `addontemplate` in the  
  [addon documentation](https://open-cluster-management.io/developer-guides/addon/#build-an-addon-with-addon-template).
- Singleton agent mode: users can now choose to start the agent as a single pod using the `Singleton` mode in the klusterlet.
```

File: content/en/community/releases.md [hunk 3]
```diff
@@ -3,7 +3,31 @@ title: Releases
...
+- ManagedClusterSet/ManagedClusterSetBinding v1beta1 API is removed.
+
+### Core components
+- ocm v0.12.0 [changelog](https://github.com/open-cluster-management-io/ocm/releases/tag/v0.12.0)
+- clusteradm  v0.7.0 [changelog](https://github.com/open-cluster-management-io/clusteradm/blob/v0.6.0/CHANGELOG.md)
+
+### Addons
+TBD
```
Comment (dhaiducek) [inline content/en/community/releases.md:29]:
```suggestion
- config-policy-controller v0.12.0 [changelog](https://github.com/open-cluster-management-io/config-policy-controller/releases/tag/v0.12.0)
- governance-policy-framework-addon v0.12.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-framework-addon/releases/tag/v0.12.0)
- governance-policy-propagator v0.12.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-propagator/releases/tag/v0.12.0)
- governance-policy-addon-controller v0.12.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-addon-controller/releases/tag/v0.12.0)
```

File: content/en/community/releases.md [hunk 4]
```diff
@@ -3,7 +3,31 @@ title: Releases
...
+- Compoenent consolidation: we made a big code refactor to merge code in regstraiton, work, placement and registration-operator in to the `ocm` repo.
+  The original separeted code repos are currently used for maintain old release only. This code consolidation allows us to build more robust e2e tests, and build a single
+  agent binary to reduce the footprint in managed cluster.
+- Addon Template API: A new `addontemplate` API is introduced to ease the development of addons. User will not need to write code and run an addon-manager
+  controller on the hub cluster. Instead, they only need to define the `addontemplate` API to create an addon. `clusteradm` also has a new command line
+  `clusteradm addon create ...` to create an addon from resource manifests files. See more details about `addontemplate` 
+  [here](https://open-cluster-management.io/developer-guides/addon/#build-an-addon-with-addon-template)
+- Singleton agent mode: user can now choose to start agent as a single pod using the `Singleton` mode in the klusterlet.
+- ManagedClusterSet/ManagedClusterSetBinding v1beta1 API is removed.
+
```
Comment (dhaiducek) [inline content/en/community/releases.md:23]:
```suggestion
- `Policy`: Add the `informOnly` option to `remediationAction` in `ConfigurationPolicy`, signaling that the remediation action set on the `Policy` should not override the `ConfigurationPolicy`'s remediation action.
- `Policy`: Security, performance, and stability improvements in controllers on both the hub and managed clusters.

```

File: content/en/community/releases.md [hunk 5]
```diff
@@ -3,7 +3,31 @@ title: Releases
 weight: -20
 ---
 
-Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.11.0`. Continue reading to view upcoming releases:
+Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.12.0`.
+Continue reading to view upcoming releases:
+
+## `0.12.0`, 11, Oct. 2023
```
Comment (dhaiducek) [inline content/en/community/releases.md:9]:
```suggestion
## `0.12.0`, 11 Oct. 2023
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Thanks for creating this PR! I have some wording suggestions on current content as well as Policy additions.

File: content/en/community/releases.md (hunk 3 above)
Comment (mikeshng) [reply content/en/community/releases.md:29]:
```
- multicloud-operators-subscription v0.12.0 [release note](https://github.com/open-cluster-management-io/multicloud-operators-subscription/releases/tag/v0.12.0)
- multicloud-operators-channel v0.12.0 [release note](https://github.com/open-cluster-management-io/multicloud-operators-channel/releases/tag/v0.12.0)
- multicloud-integrations v0.12.0 [release note](https://github.com/open-cluster-management-io/multicloud-integrations/releases/tag/v0.12.0)
```
Please add app addons as well.

Comment (mikeshng) [reply content/en/community/releases.md:29]: ^ Added app addons as well in the above suggestion.

File: content/en/community/releases.md (hunk 4 above)
Comment (mikeshng) [reply content/en/community/releases.md:23]: ^ Added ClusterPermission to the above suggestion.

**Comment by qiujian16** [comment]
> @qiujian16 for managed-serviceaccount addon:
> 
> * we added a new version v1beta1 for managed service account API: [add v1beta1 managed service account managed-serviceaccount#78](https://github.com/open-cluster-management-io/managed-serviceaccount/pull/78)
> * support using addon template to deploy agent: [support using addon template to deploy agent managed-serviceaccount#69](https://github.com/open-cluster-management-io/managed-serviceaccount/pull/69)

@zhujian7 could you propose a change on the existing PR? thanks!

File: content/en/community/releases.md [hunk 6]
```diff
@@ -3,7 +3,42 @@ title: Releases
...
+
+### Addons
+- config-policy-controller v0.12.0 [changelog](https://github.com/open-cluster-management-io/config-policy-controller/releases/tag/v0.12.0)
+- governance-policy-framework-addon v0.12.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-framework-addon/releases/tag/v0.12.0)
+- governance-policy-propagator v0.12.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-propagator/releases/tag/v0.12.0)
+- governance-policy-addon-controller v0.12.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-addon-controller/releases/tag/v0.12.0)
+- multicloud-operators-subscription v0.12.0 [changelog](https://github.com/open-cluster-management-io/multicloud-operators-subscription/releases/tag/v0.12.0)
+- multicloud-operators-channel v0.12.0 [changelog](https://github.com/open-cluster-management-io/multicloud-operators-channel/releases/tag/v0.12.0)
+- multicloud-integrations v0.12.0 [changelog](https://github.com/open-cluster-management-io/multicloud-integrations/releases/tag/v0.12.0)
+
```
Comment (zhujian7) [inline content/en/community/releases.md:41]:
```suggestion
- managed-serviceaccount v0.4.0 [changelog](https://github.com/open-cluster-management-io/managed-serviceaccount/releases/tag/v0.4.0)
```

**Comment by qiujian16** [comment]
all is resolved @mikeshng would you like to take another look and tag it?

File: content/en/community/releases.md [hunk 7]
```diff
@@ -3,7 +3,42 @@ title: Releases
...
 ---
 
-Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.11.0`. Continue reading to view upcoming releases:
+Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.12.0`.
+Continue reading to view upcoming releases:
+
+## `0.12.0`, 11 Oct. 2023
+
+The Open Cluster Management team is proud to announce the release of OCM v0.12.0! We have made architecture refactors and added several features in
+thie release:
```
Comment (dhaiducek) [inline content/en/community/releases.md:12]:
```suggestion
this release:
```

File: content/en/community/releases.md [hunk 8]
```diff
@@ -3,7 +3,42 @@ title: Releases
...
+
+- Component consolidation: we made a big code refactor to merge code in registraton, work, placement and registration-operator into the [ocm](https://github.com/open-cluster-management-io/ocm) repo.
```
Comment (dhaiducek) [inline content/en/community/releases.md:14]:
```suggestion
- Component consolidation: we made a big code refactor to merge code in registration, work, placement and registration-operator into the [ocm](https://github.com/open-cluster-management-io/ocm) repo.
```

File: content/en/community/releases.md (hunk 6 above)
Comment (dhaiducek) [reply content/en/community/releases.md:41]: I don't see this `managed-serviceaccount` addition in `content/en`, only `content/zh`?

**Review by dhaiducek — CHANGES_REQUESTED** [review]

File: content/en/community/releases.md (hunk 6 above)
Comment (zhujian7) [reply content/en/community/releases.md:41]: yes @qiujian16 please also add it into `content/en`

**Comment by mikeshng** [comment]
> all is resolved @mikeshng would you like to take another look and tag it?

Oops. I used the GitHub UI to commit suggestion and now it's failing DCO. Could you help force push please? Thanks.

**Comment by qiujian16** [comment]
updated

**Review by mikeshng — APPROVED** [review]
/approve

/lgtm

File: content/en/community/releases.md (hunk 8 above)
Comment (dhaiducek) [inline content/en/community/releases.md:14]:
```suggestion
- Component consolidation: we made a big code refactor to merge code in registration, work, placement and registration-operator into the [ocm](https://github.com/open-cluster-management-io/ocm) repo.
```

File: content/en/community/releases.md (hunk 6 above)
Comment (dhaiducek) [reply content/en/community/releases.md:41]: @qiujian16 The `managed-serviceaccount` is still missing from `content/en`

**Comment by dhaiducek** [comment]
Sorry, I wasn't fast enough 😢

**Comment by dhaiducek** [comment]
I've created a followup PR to address the review comments:
- https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/pull/390

**Pull request merged**